### PR TITLE
Rake task to create new minor version

### PR DESCRIPTION
**Pull request by indirect**
The localized sources were missed for version 1.15, leading to 404s on guides. It seems good to create a task that makes sure guides will be available for new versions. 👍

### CODE REVIEW

File: lib/tasks/versions.rake [hunk 1]
```diff
@@ -4,3 +4,15 @@ desc "Print the Bundler versions the site documents"
...
+  desc "Prepare a new version"
+  task :create do
+    last = VERSIONS.last
+    succ = VERSIONS.last.succ
+    puts "Latest version is #{last}. Creating version #{succ}..."
+    cp_r "source/#{last}", "source/#{succ}"
+    cp_r "source/localizable/#{last}", "source/localizable/#{succ}"
+    sh "middleman article 'Bundler #{succ}'"
+  end
+end
```
Comment (segiddins) [inline lib/tasks/versions.rake]: missing trailing newline

File: lib/tasks/versions.rake [hunk 2]
```diff
@@ -4,3 +4,15 @@ desc "Print the Bundler versions the site documents"
...
 end
+
+namespace :versions do
+  desc "Prepare a new version"
+  task :create do
+    last = VERSIONS.last
+    succ = VERSIONS.last.succ
+    puts "Latest version is #{last}. Creating version #{succ}..."
+    cp_r "source/#{last}", "source/#{succ}"
+    cp_r "source/localizable/#{last}", "source/localizable/#{succ}"
```
Comment (segiddins) [inline lib/tasks/versions.rake]: might also want to clear out the `what's new` page

File: lib/tasks/versions.rake [hunk 3]
```diff
@@ -4,3 +4,15 @@ desc "Print the Bundler versions the site documents"
 task :versions do
   puts VERSIONS.join(' ')
 end
+
+namespace :versions do
+  desc "Prepare a new version"
```
Comment (segiddins) [inline lib/tasks/versions.rake]: `minor version`

**Comment by indirect** [comment]
@segiddins I think I fixed everything, re-review?

**Pull request merged**